### DISCUSSION TITLE
維護期間因好奇心產生的弱智問題

### DISCUSSION THREAD

**Discussion by SQDFeng** [sharkycorps/FFXIVSharkyPatcher#3] [Q&A] [answered]
因為CN還沒更新，所以暫且沒有使用，請問說明文件裡放在XIVLauncherCN相同安裝目錄是指和XIVLauncherCN文件夾相同的位置[C:\XIVLauncherCN和C:\補丁]，還是說放進XIVLauncherCN文件夾內，與XIVLauncherCN.exe文件相同的位置[C:\XIVLauncherCN\XIVLauncherCN.exe和C:\XIVLauncherCN\補丁]（可能這個問題很弱智）

**Comment by HirotaZX** [comment, marked as answer]
是與 `XIVLauncherCN.exe` 相同的位置，不過是最外層的，不是 `app-版本號` 目錄裡的

另外 Dalamud CN 未必會在開服後立即推送新版本，建議等其能正常工作後再使用此補丁
